From c92948a4ceb181b183e2a10e869cc58b70cab280 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Mon, 8 Jun 2020 19:35:35 +0800
Subject: [PATCH 9/9] driver: soc: imx: adjust the booting order of SoC related
 code

In soc-imx8.c, there are some interfaces using IPC, so put it
behind IPC related code in Makefile so that soc-imx8.c is
built after IPC related code and make sure kernel boots up normally.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/imx/Makefile b/drivers/soc/imx/Makefile
index d49d64f1e75c..a7b9671fbadd 100644
--- a/drivers/soc/imx/Makefile
+++ b/drivers/soc/imx/Makefile
@@ -1,10 +1,10 @@
 # SPDX-License-Identifier: GPL-2.0-only
 obj-$(CONFIG_HAVE_IMX_GPC) += gpc.o
 obj-$(CONFIG_IMX_GPCV2_PM_DOMAINS) += gpcv2.o
-obj-$(CONFIG_HAVE_IMX8_SOC) += soc-imx8.o
 obj-$(CONFIG_HAVE_IMX_MU) += mu/
 obj-$(CONFIG_HAVE_IMX_SC) += sc/
 obj-$(CONFIG_HAVE_IMX_SC_PD) += pm-domains.o
 obj-$(CONFIG_HAVE_IMX_BUSFREQ) += busfreq-imx8mq.o 
+obj-$(CONFIG_HAVE_IMX8_SOC) += soc-imx8.o
 obj-$(CONFIG_HAVE_IMX_GPC_PSCI) += gpc-psci.o
 obj-$(CONFIG_HAVE_IMX8_SECO) += seco_mu/
-- 
2.17.1

